src/control/types.ts

```diff
@@ -82,3 +37,18 @@ export type PodType =
   | 'p2.x4'
   | 'p2.x8'
   | string;
+
+export const ValidPodTypes: PodType[] = [
```

Comment:
It's a bit odd that the `PodType` type includes all these values but then also just `string` 🤔

Reply:
Yeah that was a holdover from before.... I'm not sure why. I'd totally be pro removing it